app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -332,6 +331,22 @@ private void sendForms(FormRecord[] records) {
         }
     }
 
+    private static void logSubmissionAttempt(FormRecord record) {
+        String attemptMesssage = String.format(
+                "Attempting to submit form with id %1$s and submission ordering number %2$s",

```

Comment:
the submission order here feels a little odd to include, is it primarily for debugging the dev work? might not be needed long term (and might introduce some amount of confusion).